auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -10,7 +10,7 @@
       v-model="dialog"
       persistent
       scrollable
-      max-width="675"
+      max-width="50rem"
```

Comment:
To allow business id column to fit without wrapping, ie.

![image](https://github.com/bcgov/sbc-auth/assets/10002889/af95deb8-bd00-4f3d-8d0b-f2f4a856e5de)